src/app/sections/ProbeDetails.svelte

```diff
@@ -122,26 +170,53 @@ h2 {
     {/if}
     <div class=drawer-section>
         {#if $store.probe.description}
-            <h2 class=detail__heading--01>description</h2>
-            <div class='probe__description helper-text--01'>
+            <h2 class="detail__heading--01">description</h2>
+            <div class="probe-description helper-text--01">
                 {@html $store.probe.description}
             </div>
         {/if}
     </div>
     {#if $store.probe.bugs && $store.probe.bugs.length}
     <div class="drawer-section">
-            <h2 class=detail__heading--01>associated bugs</h2>
-            <div class="bug-list helper-text--01">
-            {#each $store.probe.bugs as bugID, i (bugID)}
-                <a
-                href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
-                target="_blank">{bugID}</a>
-            {/each}
-            </div>
+        <h2 class="detail__heading--01">associated bugs</h2>
+        <div class="bug-list helper-text--01">
+        {#each $store.probe.bugs as bugID, i (bugID)}
+            <a
+            href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
+            target="_blank">{bugID}</a>
+        {/each}
+        </div>
     </div>
     {/if}
-    <div class='drawer-section drawer-section--end'>
-        <h2 class=detail__heading--01>export</h2>
+    <div class="drawer-section">
+        <h2 class="detail__heading--01">extras</h2>
+        <dl class="details-list">
+        {#if $store.probe.active !== undefined}
+            <dt><span class="label label-text--01 label--definition-title">active</span></dt>
+            <dd class="details-list-definition label-text--01">{$store.probe.active}</dd>
```

Comment:
Furthermore, I think we should probably think of breaking out labels into two distinct things: normal (which is what we have here) and compact. Compact (which I think we should change this to) should probably have much small left / right padding, and the text size should be smaller than the normal one. Otherwise it's hard to know what my eyes are supposed to make of this visual hierarchy. I'm open to removing the label entirely and just using overline styling or something like that.

Reply:
More to this point, it'd be great if we could just say `inactive` here, and make the label bg light `--pantone-red-xxx` with darker text `--pantone-red-700` or something like that. "active: false" feels like too direct mapping from the data to the frontend, and I think people would just appreciate "inactive". Same probably goes with prerelease – maybe we just say "unavailable in prerelease" or similar copy

Reply:
Really good points. I wasn't sure of the color plan moving forward so I just picked something random but more importantly I really agree with just "active" vs "inactive" rather than writing out the booleans.

Reply:
no worries re: the color – I haven't communicated much of that! I'm thinking this would be probably a good use of Storybook to demonstrate / agree on the colors for GLAM, actually. That way we can come to a consensus about these details that currently are unlocked in my head, unchallenged.